PROTOCOL.md

```diff
@@ -149,17 +150,21 @@ represented by a space), and the rest of the string being their username.
 
 > Changes the HTML display of the `|uhtml|` message named (NAME).
 
-`|join|USER` or `|j|USER`
+`|join|USER@STATUS` or `|j|USER@STATU`
 
-> `USER` joined the room.
+> `USER` joined the room. Optionally, `USER` can end in `@` followed by a user status message.
+> A `STATUS` starting in `!` indicates the user is away.
 
-`|leave|USER` or `|l|USER`
+`|leave|USER@STATU` or `|l|USER@STATU`
 
-> `USER` left the room.
+> `USER` left the room. Optionally, `USER` can end in `@` followed by a user status message.
+> A `STATUS` starting in `!` indicates the user is away.
```

Comment:
I think we need to explain what 'away' means. Furthermore, if `!` indicates 'away' (in the same way we originally envisioned a name ending with `@` to mean), it presumably means we'd want to support distinctions between `!idle` / `!afk` / `!brb` / `!bbl` / `!away` etc, in which case, I'd like to see the *manual* `/away` command trigger a different 'away' status than simply being marked inactive (eg. `!away` vs. `!idle`).

Alternatively, just drop the `!` and we can just say `'@away'` means someone is 'away' in our previous sense.

Reply:
I did that